### PR TITLE
Dev branch

### PR DESCRIPTION
**Pull request by Nick5500**
module ES6 added

### CODE REVIEW

File: server/controllers/movie.js [hunk 1]
```diff
@@ -44,11 +34,7 @@ class MovieController {
 
   static async updateMovie(req, res) {
     try {
-      const id = req.params.id;
-
-      await Movie.update(req.body, {
-        where: { id }
-      })
+      await movieService.updateMovie(req.params.id, req.body)
```
Comment (MaxLogvyniukFive) [inline server/controllers/movie.js:37]: Does this method not return any result?

File: server/service/movie.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        time,
+        rating
+      })
+    } catch (e) {
+      console.log(e)
+    }
+  };
+
+  static async updateMovie(movieId, data) {
+    try{
```
Comment (MaxLogvyniukFive) [inline server/service/movie.js:38]: Please setup linter
`try  {`

[https://medium.com/@oxodesign/node-js-express-with-typescript-eslint-jest-prettier-and-husky-part-2-f129188ce404](https://medium.com/@oxodesign/node-js-express-with-typescript-eslint-jest-prettier-and-husky-part-2-f129188ce404)

File: server/controllers/movie.js [hunk 3]
```diff
@@ -13,11 +13,7 @@ class MovieController {
 
   static async getMovieById(req, res) {
     try {
-      const id = req.params.id;
-
-      const oneMovie = await Movie.findAll({
-        where: { id }
-      })
+      const oneMovie = await movieService.getOneMovie(req.params.id)
```
Comment (MaxLogvyniukFive) [inline server/controllers/movie.js:17]: Need add check do we review error or result

File: server/controllers/movie.js [hunk 4]
```diff
@@ -27,13 +23,7 @@ class MovieController {
...
-      const { name, time, rating } = req.body
-
-      const createdMovie = await Movie.create({
-        name,
-        time,
-        rating
-      })
+      const createdMovie = await movieService.createMovie(req.body);
 
       res.json(createdMovie)
```
Comment (MaxLogvyniukFive) [inline server/controllers/movie.js:28]: Use the same approach for response building - [https://github.com/max-logvyniuk/et-chat/blob/master/api/server/src/utils/Utils.js](https://github.com/max-logvyniuk/et-chat/blob/master/api/server/src/utils/Utils.js)
[https://github.com/max-logvyniuk/et-chat/blob/master/api/server/src/controllers/UserController.js#L20](https://github.com/max-logvyniuk/et-chat/blob/master/api/server/src/controllers/UserController.js#L20)

File: server/controllers/movie.js (hunk 1 above)
Comment (MaxLogvyniukFive) [reply server/controllers/movie.js]: @Nick5500 please answer the question)

Comment (Nick5500) [reply server/controllers/movie.js]: @MaxLogvyniukFive This method returns 1 or 0

File: server/service/movie.js (hunk 2 above)
Comment (Nick5500) [reply server/service/movie.js:38]: @MaxLogvyniukFive Yes, I applied eslinter in IDE, It works

File: server/.eslintrc.cjs [hunk 5]
```diff
@@ -0,0 +1,141 @@
+module.exports = {
```
[the remark this answers is not in the document]
Comment (Nick5500) [reply server/.eslintrc.cjs:1]: @MaxLogvyniukFive When I initialize eslint, It automatically creates file with this extention - ".cjs". I tried to change file's extention but It doesn't work with extention ".js".

**Review by MaxLogvyniukFive — APPROVED** [review]

**Pull request merged**